kratos/includes/geometry_container.h

```diff
@@ -0,0 +1,407 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Pooyan Dadvand
+//
+//
+
+
+#if !defined(KRATOS_GEOMETRY_CONTAINER_H_INCLUDED )
+#define  KRATOS_GEOMETRY_CONTAINER_H_INCLUDED
+
+
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/define.h"
+#include "containers/pointer_hash_map_set.h"
+#include "containers/data_value_container.h"
+
+
+namespace Kratos
+{
+
+///@name Kratos Classes
+///@{
+
+template<class TGeometryType>
+class GeometryContainer : public DataValueContainer, public Flags
```

Comment:
since it is a container I would move it to the `containers`-folder

Reply:
Sure, makes sense. I was adapting it from mesh, thus it stayed in includes.